codegen/rustjit.rs

```diff
@@ -17,16 +17,30 @@ pub struct RustjitContext {
 impl RustjitContext {
     pub fn get(&self, value: IrValue) -> u64 {
         match value {
-            IrValue::Variable(ty, id) => todo!(),
-            IrValue::Register(ty, id) => todo!(),
-            IrValue::Constant(constant) => todo!(),
+            IrValue::Variable(_ty, id) => *self.variables.get(&id).unwrap(),
+            IrValue::Register(ty, id) => bytes_to_u64(self.registers.read(id), ty),
+            IrValue::Constant(constant) => match constant {
+                IrConstant::U8(value) => value as u64,
+                IrConstant::U16(value) => value as u64,
+                IrConstant::U32(value) => value as u64,
+                IrConstant::U64(value) => value as u64,
+                IrConstant::I8(value) => value as u64,
+                IrConstant::I16(value) => value as u64,
+                IrConstant::I32(value) => value as u64,
+                IrConstant::I64(value) => value as u64,
+            },
         }
     }
+
     pub fn set(&mut self, value: IrValue, data: u64) {
         match value {
-            IrValue::Variable(ty, id) => todo!(),
-            IrValue::Register(ty, id) => todo!(),
-            IrValue::Constant(constant) => todo!(),
+            IrValue::Variable(_, id) => {
+                self.variables.insert(id, data);
+            }
+            IrValue::Register(_, id) => {
+                self.registers.write(id, &data.to_le_bytes());
+            }
+            IrValue::Constant(_) => panic!("Contant cannot be set"),
```

Comment:
Should we only accept `Variable` or `Register`  here?

Reply:
Yes, I think set `Constant` doesn't make sense.